sdk/src/main/java/com/hedera/hashgraph/sdk/Network.java

```diff
@@ -36,13 +41,114 @@
 
         try {
             setNetwork(network);
-        } catch (InterruptedException e) {
-            // This should never occur. The network is empty.
-        } catch (TimeoutException e) {
+        } catch (InterruptedException | TimeoutException e) {
             // This should never occur. The network is empty.
         }
     }
 
+    static Network forNetwork(ExecutorService executor, Map<String, AccountId> network) {
+        return new Network(executor, network);
+    }
+
+    static Network forMainnet(ExecutorService executor) {
```

Comment:
Exposing a way to switch these built-in endpoints to the TLS port without requiring clients to configure every IP would be nice.

Reply:
Yea, you're definitely right. For this release we wanted to get as basic of TLS support as humanly possible which is why everything looks bad is is hard coded.